core/src/main/java/nl/pim16aap2/bigDoors/moveBlocks/BridgeMover.java

```diff
@@ -223,67 +223,73 @@ private void createAnimatedBlocks()
                     Location newFBlockLocation = new Location(world, xAxis + 0.5, yAxis, zAxis + 0.5);
 
                     Block vBlock = world.getBlockAt(xAxis, yAxis, zAxis);
-                    Material mat = vBlock.getType();
-                    if (Util.isAllowedBlock(mat))
-                    {
-                        byte matData = vBlock.getData();
-                        BlockState bs = vBlock.getState();
-                        MaterialData materialData = bs.getData();
+                    int finalYAxis = yAxis;
+                    int finalZAxis = zAxis;
+                    int finalXAxis = xAxis;
+                    double finalRadius = radius;
+                    BigDoors.getScheduler().runTask(vBlock.getLocation(), () -> {
```

Comment:
See the note above.

Creating the falling blocks in 1 task feels much cleaner than scheduling potentially thousands of tasks just to create a single door.